understanding/20/error-identification.html

```diff
@@ -70,11 +69,7 @@ <h2>Intent of Error Identification</h2>
          error information to the user. For example, certain technologies can specify that
          the user's input must not fall outside a specific range, or that a form field is required.
          Currently, few technologies support this kind of programmatic information, but this
-         Success Criterion does not require, nor prevent it. However, other Success Criteria may apply - for
-         instance, the programmatic identification of form controls/inputs that are in error is covered
-         more specifically in <a href="name=role-value">4.1.2 Name, Role, Value</a>, and whether or not
-         error messages are programmatically linked to their respective form controls/inputs is covered
-         by <a href="info-and-relationships">1.3.1 Info and Relationships</a>.
+         Success Criterion does not require, nor prevent it. However, other Success Criteria may apply such as <a href="name=role-value">4.1.2 Name, Role, Value</a>, and <a href="info-and-relationships">1.3.1 Info and Relationships</a>.
```

Comment:
would be nice to hint at how those other SCs apply, directly here, rather than just handwaving...

Reply:
We could either continue quite a long discussion about that (@WilcoFiers disagreed on the face of it), or get the current changes in and deal with that separately.

Reply:
maybe worth adding a tiny bit of context on these sorts of changes to PRs then, as they otherwise seem quite arbitrary

Reply:
Yep, adding that to a new survey item.

Reply:
We should make it clear that if an image is used the image itself must communicate the errors also communicate in alt text.  We don't want people to stuff extra needed details into alt text with only a visual image that doesn't communicate the error and what is wrong.  This could be misused if we don't clarify this.

Reply:
I agree that a self evident error would be ok with regards to required fields.  For example, if a required field is empty and has an error icon then it's ok as it's required and an icon is sufficient in that case to communicate the fact that a required field was not filled out.  For other errors though like incorrect data format it's a different story.

Reply:
(a small side note here @mraccess77 - you're replying to a sub-thread about the change @alastc made to take our explicit references to other SCs that may also apply, based on @WilcoFiers's feedback at the time. your response, I believe, relate to the main discussion thread here)